website/docs/cassandra-pod-delete.md

```diff
@@ -59,7 +59,7 @@ sidebar_label: Cassandra Pod Delete
 
 #### Sample Rbac Manifest
 
-[embedmd]: # "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/cassandra/cassandra-pod-delete/rbac.yaml yaml"
+[embedmd]: # "https://raw.githubusercontent.com/litmuschaos/chaos-charts/v1.10.x/charts/cassandra/cassandra-pod-delete/rbac.yaml yaml"
```

Comment:
Why do we need to change the version in the main docs folder.  Should it be on versioned docs only?

Reply:
It was there in litmus-docs as well. So i added here as well.